app/lib/generator.js

```diff
@@ -15,8 +15,16 @@ exports.generate = async () => {
   if (!_.get(latestTag, "commit.committed_date") || !_.get(secondLatestTag, "commit.committed_date")) throw new Error(`Cannot find latest and second latest tag. Abort the program!`);
   const startDate = _.get(secondLatestTag, "commit.committed_date");
   const endDate = _.get(latestTag, "commit.committed_date");
-  const changeLog = await ChangelogLib.getChangelogByStartAndEndDate(startDate, endDate);
+
+  // allow the end date to be adjusted by a few seconds to catch issues that are automatially closed by
+  // a MR and are time stamped a few seconds later.
+  Logger.debug(`EndDate:      ${endDate}`);
+  const parsedDate = new Date(Date.parse(endDate))
```

Comment:
The code block you added works but could I suggest to use Moment Timezone, please. I would like to use the moment library for time manipulation in the project. It's slightly easy to manage than the pure javascript Date object

```
let endDate = _.get(latestTag, "commit.committed_date");
if (Env.ISSUE_CLOSED_SECONDS > 0) {
    Logger.debug(`EndDate:      ${endDate}`);
    endDate = Moment.tz(endDate, Env.TZ).add(Emv.ISSUE_CLOSED_SECONDS, "seconds");
    Logger.debug(`New End Date: ${newEndDate}`);
}
```

Reply:
Code updated per your suggestions.